common/src/main/scala/com/gu/recipeasy/db/DB.scala

```diff
@@ -164,10 +164,22 @@ class DB(contextWrapper: ContextWrapper) {
     }
   }
 
+  def countRecipes(): Long = {
+    contextWrapper.dbContext.run(quote(query[Recipe])).size
+  }
+
   def countRecipesInGivenStatus(status: Status): Long = {
     contextWrapper.dbContext.run(quote(query[Recipe]).filter(r => r.status == lift(status.toString))).size
   }
 
+  def curationCompletionRatio(): Double = {
+    (countRecipesInGivenStatus(Pending) + countRecipesInGivenStatus(Curated)) / countRecipes()
+  }
+
+  def verificationCompletionRatio(): Double = {
+    (countRecipesInGivenStatus(Pending) + countRecipesInGivenStatus(Curated) + countRecipesInGivenStatus(Verified) + countRecipesInGivenStatus(Finalised)) / countRecipes()
```

Comment:
Again, it's nothing to worry about right now, but worth being aware that you are making duplicate calls to `countRecipesInGivenStatus(Pending)`  and `countRecipesInGivenStatus(Curated)` when it's not really necessary making it sub optimal.